### PR TITLE
Free Germans saving percents in the App Store

### PR DESCRIPTION
**Pull request by DonSqueak**
Free searches for free Germans ;)

### CODE REVIEW

File: CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff [hunk 1]
```diff
@@ -1118,7 +1118,7 @@
         <note/>
       </trans-unit>
       <trans-unit id="Save %@" xml:space="preserve">
-        <source>Save %@</source>
+        <source>Save %@</source><target>Spare %@</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:1121]: Still dunno where "Yearly Subscription" (just in front of Save 17 %) is coming from. We have "Yearly" in the file, but that's seemingly a whole different string...

**Comment by DonSqueak** [comment]
Added first draft of German iOS description @cliss @technocidal

**Comment by DonSqueak** [comment]
Guess how long before I decided to run a diff ob iOS v. visionOS? -- hint: too f***ing long :D

File: CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff (hunk 1 above)
Comment (cliss) [reply CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:1121]: You're right; I need to change `Yearly Subscription` in App Store Connect → `Yearly` for consistency's sake
Screenshot 2024-05-14 at 5 54 15 PM

Comment (DonSqueak) [reply CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:1121]: Aaaaah, now I get it

**Comment by DonSqueak** [comment]
@cliss Casey, do you by any chance know why I have a gazillion commits already filling up my PRs, when I open them, all of which were already merged into upstream?

After you merge a PR, I always update my fork, and start with a number of commits already in the branch. What is going on there... I wonder...

image

**Comment by technocidal** [comment]
@DonSqueak Looking at the shape of the commit graph, it seems that @cliss started squashing merges so the information about some of the commits that were merged gets lost. That's probably also why on your fork it says that it is `12 commits ahead`. Sadly updating your fork just transports over the actual squashed merge commit but not the information which commits have been squashed.

A way around this issue would be to always create a new fork which might be cumbersome. There is probably also a way of rebasing your main branch after a merge to take the exact shape of @cliss main branch.

**Comment by DonSqueak** [comment]
Ah, that makes sense. Well, as long as the zombie commits don’t actually hurt anyone, I may just drag them along. Or I’ll try a rebase. Thanks!

**Comment by technocidal** [comment]
It should be fine, however the automatic commit messages that Github creates for the squash always contains all commits mentioned in the PR. That means, that it will get longer and longer which means that at some point it will be meaningless. You can already see this when comparing the [commit message](https://github.com/cliss/callsheet-localizations/commit/4998519bcdd64805b06b4aa6920b97c4d268e94e) of #20 and the [commit message](https://github.com/cliss/callsheet-localizations/commit/1c282588e2c4850de0b1dfadf8f0fdad2a4e0d33) of #22.

AFAIK it should not result in weird merge conflicts down the line.

File: AppStore/German/Description.iOS.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Wolltest du schon immer solche Fragen beantworten können, *ohne* alle fünf Sekunden irgendwo aufgefordert zu werden dich anzumelden? Ohne Werbung und ohne automatisch abgespielte Videos? Ohne auf ein multinationales Konglomerat angewiesen zu sein, das dir Dinge verkaufen möchte?
+
+Dann wirst du Callsheet lieben.
+
+Callsheet ist der beste Weg, um schnell Informationen über Besetzung und Crew in Fernsehserien und Filmen zu finden. Es geht mit deiner Zeit respektvoll um, versucht nicht, dir Dinge zu verkaufen, die du nicht brauchst, und ist darauf ausgelegt, deine Fragen schnell und unkompliziert zu beantworten.
+
+Callsheet bietet auch einige neuartige Funktionen, die deine bisherige Film-/Seriendatenbank-App nicht hat:
+
+• Spoiler vermeiden beim Anschauen von TV-Serien, indem optional Charakternamen, Episodenzahlen, Episodentitel oder Episodenvorschaubilder ausgeblendet werden.
```
Comment (technocidal) [inline AppStore/German/Description.iOS.txt:11]: Looking at this long-form description I'm starting to feel like "TV-Serien" might be better than "TV-Sendungen" in general.

File: AppStore/German/Description.iOS.txt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+• Leichter Zugriff auf das Alter der Cast-Mitglieder *bei der Veröffentlichung eines Films* direkt neben der Besetzungsliste
+• Unkomplizierte Ansicht des Alters einer Person neben jedem Eintrag in ihrer Filmografie
+• Direkte Anzeige der Körpergröße der Schauspielenden (wenn möglich)
+• Unterstützt den Dunkelmodus
+• Unterstützt VoiceOver
+• Alternative App-Symbole für Abonnenten
+• Automatische Integration mit der wunderbaren Channels-App - keine Anmeldung erforderlich!
+• Experimentelle Integration mit Plex, ebenfalls ohne Anmeldung.
+
+Callsheet wird von einer Person geschrieben, die WIRKLICH darauf bedacht ist, deine Zeit zu respektieren. Callsheet ist darauf ausgelegt, dass du schnell und unkompliziert das findest, was du brauchst, und dann wieder zu dem Film oder der Serie zurückkehren kannst, die dir wirklich am Herzen liegt. Keine Ablenkungen und keine Aufforderungen, dich irgendwo anzumelden.
```
Comment (technocidal) [inline AppStore/German/Description.iOS.txt:25]: Maybe replace the second Callsheet with just "Es" oder "Die App"

```...bedacht ist, deine Zeit zu respektieren. Es ist darauf ausgelegt...```

```...bedacht ist, deine Zeit zu respektieren. Die App ist darauf ausgelegt...```

Comment (technocidal) [reply AppStore/German/Description.iOS.txt:25]:
```suggestion
Callsheet wird von einer Person geschrieben, die WIRKLICH darauf bedacht ist, deine Zeit zu respektieren. Callsheet ist darauf ausgelegt, dass du schnell und unkompliziert das findest, was du suchst, und dann wieder zu dem Film oder der Serie zurückkehren kannst, die dir wirklich am Herzen liegt. Keine Ablenkungen und keine Aufforderungen, dich irgendwo anzumelden.
```

**Review by technocidal — APPROVED** [review]

File: AppStore/German/Description.iOS.txt (hunk 2 above)
Comment (DonSqueak) [reply AppStore/German/Description.iOS.txt:11]: It’s a bit murky: the main use case is indeed TV series, but there are also other shows in there, like "unscripted" "reality" TV ;)

Comment (DonSqueak) [reply AppStore/German/Description.iOS.txt:11]: Hence, for the I’m a Celebrity Get Me out of Here crowd, I left it TV-Sendungen in the UI. The quoted passage from the store description is more about TV series, so I translated it TV-Serien there

Comment (technocidal) [reply AppStore/German/Description.iOS.txt:11]: That makes sense.

File: AppStore/German/Description.iOS.txt (hunk 3 above)
Comment (DonSqueak) [reply AppStore/German/Description.iOS.txt:25]: Good suggestions!

**Comment by cliss** [comment]
@technocidal That's weird that squashing is hosing everything up, but your description makes sense. It's been _a minute_ since I've done [Git flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) at my last jobby job.

I'm all ears if there's a better/more modern approach (aka Best Practice™) that I should be following!

**Pull request merged**